core/drand_control.go

```diff
@@ -20,6 +20,9 @@ import (
 	vss "github.com/drand/kyber/share/vss/pedersen"
 )
 
+// errPreempted is returned on reshares when a subsequent reshare is started concurrently
+var errPreempted = errors.New("time out: pre-empted")
```

Comment:
Can the error message be a bit more descriptive, perhaps it should include what is in the comment above?

Reply:
note that this error is returned in testing, and is output in a context of:
```
--- FAIL: TestDrandResharePreempt (14.93s)
    drand_test.go:537: 
        	Error Trace:	drand_test.go:537
        	            				asm_amd64.s:1373
        	Error:      	Received unexpected error:
        	            	rpc error: code = Unknown desc = drand: err during DKG: drand: error from dkg: time out: pre-empted
        	Test:       	TestDrandResharePreempt
        	Messages:   	non-leader node (new?false) error during reshare 
```

Reply:
`core/drand_control.go` is not used in production?

Reply:
I believe this method is always used in the context of an rpc, so you will at least receive the 
```
rpc error: code = Unknown desc = drand: err during DKG: drand: error from dkg: time out: pre-empted
```
portion.
I think that's probably sufficient to track down what's happened, especially since this triggers when the user has manually run a second share while the first one is running (or more likely has ctrl^c'd the first share and never sees anything)